lib/commands/smapi/smapi-command-handler.js

```diff
@@ -115,8 +116,13 @@ const _showCheckStatusHint = (locationHeader) => {
         const baseCliCommand = processor.processOperationName(operationName);
         const queryStringParameters = queryString.parse(queryStringPart);
         parameters = { ...swaggerMatch.params, ...queryStringParameters };
-        const cliParams = Object.keys(parameters).map(key => `--${kebabCase(key)} ${parameters[key]}`).join(' ');
-        statusCheckHintCommand = `ask smapi ${baseCliCommand} ${cliParams}`;
+        const cliParams = Object.keys(parameters).map(key => {
+            // mapping custom command option if exists
+            const commandOption = kebabCase(apiToCommanderMap.get(key) || key);
+            return `--${commandOption} ${parameters[key]}`;
```

Comment:
Looking through these customization cases, so far there are three types: `skipUnwrap`, `name`, `skip`, and the `customParameters`. Based on this use case, I believe `skip` should also be considered, i.e. the vendorId should be prevented to be displayed in the hint string.

Reply:
Using current latest version, I can verify the vendorId is still part of the string based on the following test:
```
$ ask smapi submit-skill-for-certification -s amzn1.ask.skill.12345 --publication-method MANUAL_PUBLISHING -p pub
[Warn]: This is an asynchronous operation. Check the progress using the following command: ask smapi list-skills-for-vendor --vendor-id MWD3O0I5T2G52 --skill-id amzn1.ask.skill.12345
Command executed successfully!
```


Please also add a test for this case to make sure similar case can be handled well.